server/src/main/java/org/spine3/server/stand/StandStorage.java

```diff
@@ -69,4 +72,24 @@ protected StandStorage(boolean multitenant) {
      */
     public abstract ImmutableCollection<EntityRecord> readAllByType(TypeUrl type,
                                                                     FieldMask fieldMask);
+
+    /**
+     * The {@code StandStorage} is specified not to satisfy the {@link EntityQuery}.
```

Comment:
As discussed vocally, let's modify this Javadoc section, so that it has the general description, the equivalence statement you have below. And only then this thought, but explained in much more words, so that other people understand what it means.